geomesa-convert/geomesa-convert-xml/pom.xml

```diff
@@ -20,6 +20,11 @@
 
     <dependencies>
         <dependency>
+            <groupId>net.sf.saxon</groupId>
+            <artifactId>Saxon-HE</artifactId>
+            <version>9.7.0-7</version>
```

Comment:
this could be a problem:
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=5368
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=1903

@jnh5y think it's still worth opening a CQ?